src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,69 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURES_COUNTER = 5;
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+
+    public Figure getRandomFigure() {
+        switch (random.nextInt(FIGURES_COUNTER)) {
+            case 0: return getRandomSquare();
+            case 1: return getRandomRectangle();
+            case 2: return getRandomCircle();
+            case 3: return getRandomRightTriangle();
+            case 4: return getIsoscelesTrapezoid();
+            default: return getIsoscelesTrapezoid();
```

Comment:
```suggestion
            default: return getDefaultFigure();
```

мне кажется так было бы логичней) Но в общем можешь остаивть getIsoscelesTrapezoid()

Reply:
Согласен с Вами, но ориентировался на это
![image](https://user-images.githubusercontent.com/93541990/142885077-039923a5-c039-4244-a2f3-fb8bb3a17e2c.png)

Reply:
Но в общем мы тут null и не возвращаем, а возвращаем дефолтную фигуру, так что все ок)